kyma-project/busola
# Fix overlapping requests

**Testing**

1) Go to cluster overview
2) Select graph for nodes with biggest timespan
3) Select graph for CPU

CPU graph should be broken - try several times

**Broken**
<img width="615" alt="image" src="https://user-images.githubusercontent.com/10504661/157858288-1c867100-9456-4d0d-ae99-a7407f4fd0e7.png">

**Expected**
<img width="615" alt="image" src="https://user-images.githubusercontent.com/10504661/157858442-c5377155-d748-48f1-8f7f-3e174be34191.png">
